vmware_exporter/helpers.py

```diff
@@ -15,6 +27,19 @@ def batch_fetch_properties(content, obj_type, properties):
         recursive=True
     )
 
+    """
+        Gathering all custom attibutes names are stored as key (integer) in CustomFieldsManager
+        We do not want those keys, but the names. So here the names and keys are gathered to
+        be translated later
+    """
+    allCustomAttributesNames = dict(
```

Comment:
Can you add an example of what this looks like as a metric?

Reply:
I dont know what you mean but in a metric a custom attribute would look something like this:
```
vmware_vm_net_droppedTx_summation{Role="Nginx",ClientID="36310",Configurator="John Doe",Creator="Jane Doe",ApplicatioName="HelloWorld",CreationTicket="2019/000215454",cluster_name="Cluster0018",dc_name="DataCenter02",host_name="dc02cl00018host1111.domain.com",tags="n/a",vm_name="vm-nginx4781"} 0.0
```